src/panels/lovelace/hui-root.ts

```diff
@@ -206,19 +208,27 @@ class HUIRoot extends LitElement {
                       ${this._yamlMode
                         ? html`
                             <paper-item @click="${this._handleRefresh}"
-                              >Refresh</paper-item
+                              >${this.hass!.localize(
+                                "ui.panel.lovelace.editor.refresh"
+                              )}</paper-item
                             >
                           `
                         : ""}
                       <paper-item @click="${this._handleUnusedEntities}"
-                        >Unused entities</paper-item
+                        >${this.hass!.localize(
+                          "ui.panel.lovelace.editor.unused_entities"
+                        )}</paper-item
                       >
                       <paper-item @click="${this._editModeEnable}"
                         >${this.hass!.localize(
                           "ui.panel.lovelace.editor.configure_ui"
                         )}</paper-item
                       >
-                      <paper-item @click="${this._handleHelp}">Help</paper-item>
+                      <paper-item @click="${this._handleHelp}"
+                        >${this.hass!.localize(
+                          "ui.panel.lovelace.editor.help"
```

Comment:
This is not part of the editor but part of the menu.

Reply:
This was because the configure_ui key was already in the editor so I wanted to keep things as is. Will refactor it all as per our conversation.